Source/Widgets/CesiumWidget/CesiumWidget.js

```diff
@@ -144,6 +144,7 @@ function configureCameraFrustum(widget) {
  * @param {Boolean} [options.shadows=false] Determines if shadows are cast by light sources.
  * @param {ShadowMode} [options.terrainShadows=ShadowMode.RECEIVE_ONLY] Determines if the terrain casts or receives shadows from light sources.
  * @param {MapMode2D} [options.mapMode2D=MapMode2D.INFINITE_SCROLL] Determines if the 2D map is rotatable or can be scrolled infinitely in the horizontal direction.
+ * @param {Boolean} [options.blurActiveElementOnCanvasFocus=true] If set to true, the the active element will blur when the viewer's canvas is clicked. Setting this to false is useful for cases when the canvas is clicked only for retrieving georgaphical or an entity's data without actually meaning to set the canvas to be the active element.
```

Comment:
```suggestion
 * @param {Boolean} [options.blurActiveElementOnCanvasFocus=true] If true, the the active element will blur when the viewer's canvas is clicked. Setting this to false is useful for cases when the canvas is clicked only for retrieving position or entity data without actually meaning to set the canvas to be the active element.
```

Thanks, this is helpful! I tweaked this to be a bit shorter and fix a spelling typo. The comment in `Viewer` should be update to match.